### PR TITLE
Fix up several UI problems

### PR DESCRIPTION
**Pull request by jepler**
This:
 * Removes the import and use of GLUT. GLUT is never used to create a UI window by Axis or Gremlin.
 * fixes the gremlin commandline program so it starts again
 * fixes exceptions printed on the terminal while running configs/sim/touchy/gladevcp/touchy.ini which may have also affected correct behavior
 * comments on why it's necessary to make direct GLX calls in gremlin, which is obstensibly a Gtk program
 * remove some commented out code

I don't know how these fixes intersect (if at all) with the current problems running axis & gremlin-based UIs on unstable or testing.

### CODE REVIEW

File: src/emc/usr_intf/gremlin/gremlin.py [hunk 1]
```diff
@@ -115,12 +115,6 @@ def __init__(self, inifile):
     
         self.xwindow_id = None
 
-        #self.set_has_alpha(True)
-        #'set_has_stencil_buffer',
-        glutInit()
```
Comment (SebKuzminsky) [inline src/emc/usr_intf/gremlin/gremlin.py:120]: Do you want to delete line 49 too?

Comment (jepler) [reply src/emc/usr_intf/gremlin/gremlin.py:120]: Argh, yes

Comment (jepler) [reply src/emc/usr_intf/gremlin/gremlin.py:120]: Updated

**Comment by SebKuzminsky** [comment]
The build failure is my fault, #2315 should fix it.

**Comment by c-morley** [comment]
Did you guys try Qtvcp? - is this something that applies to it ? The openGL code for all three Axis, Gladevcp and Qtvcp are very similar.

**Pull request merged**